diana-api/src/main/java/org/apache/diana/api/reader/DateReader.java

```diff
@@ -0,0 +1,32 @@
+package org.apache.diana.api.reader;
+
+
+import org.apache.diana.api.ReaderField;
+
+import java.util.Date;
+
+/**
+ * Class to reads and converts Date type
+ *
+ * @author Daniel Cunha <danielsoro@apache.org>
+ */
+public final class DateReader implements ReaderField {
+
+    @Override
+    public boolean isCompatible(Class clazz) {
+        return Date.class.equals(clazz);
+    }
+
+    @Override
+    public <T> T read(Class<T> clazz, Object value) {
```

Comment:
Sounds much better to delegate it to LocalDateTime or just have this implementation on TemporalReader :)